etcdserver/config.go

```diff
@@ -54,6 +54,8 @@ type ServerConfig struct {
 
 	AutoCompactionRetention int
 	QuotaBackendBytes       int64
+	// MaxRequestBytes is the max request size that raft accepts.
```

Comment:
`MaxRequestBytes is the maximum request size to send over raft.`

raft will accept whatever is sent to it, this check is only API-level